Makefile

```diff
@@ -1,3 +1,7 @@
+lint:
+	@echo "Linting ..."
```

Comment:
The linter `loggercheck` as per [this][1] was available [golangci-lint v1.49.0][2]. However, I see it in effect only in the next version [golangci-lint v1.50.0][3] (which happens to be a major). Else, it fails with the following error:
```
Linting ...
ERRO Running error: unknown linters: 'loggercheck', run 'golangci-lint help linters' to see the list of supported linters 
make: *** [Makefile:3: lint] Error 3
```

I suggest to add a line in `make lint` something similar to: 
```
Please make sure you are on golangci-lint v1.50.0 or later:\n\tgo install github.com/golangci/golangci-lint/cmd/golangci-lint@latest\n`
```

[1]: https://golangci-lint.run/usage/linters/#disabled-by-default
[2]: https://github.com/golangci/golangci-lint/releases/tag/v1.49.0
[3]: https://github.com/golangci/golangci-lint/releases/tag/v1.50.0

Reply:
where I must to add this line? btw, it is last version go `golangci-lint v1.50.0`

Reply:
Add the line to print in the same target, `make lint`.
```
lint:
         @echo "Linting ...Please make sure you are on golangci-lint v1.50.0 or later:\n\tgo install github.com/golangci/golangci-lint/cmd/golangci-lint@latest\n`"
```
In [golangci-lint v1.50.0](https://github.com/golangci/golangci-lint/releases/tag/v1.50.0) and [latest (golangci-lint v1.50.1)](https://github.com/golangci/golangci-lint/releases/tag/v1.50.1). The issue occurs when using older versions.

Reply:
@vaguecoder done

Reply:
Please also check the 2 comments below

Reply:
done